01_stratified_classification/README.md

```diff
@@ -0,0 +1,27 @@
+# Comparing optimization methods for stratified mutation status classification
+
+Manuscript: https://greenelab.github.io/optimizer-manuscript/
+
+This (sub-)repository contains code related to our manuscript "Optimizer’s dilemma: optimization strongly influences model selection in transcriptomic prediction".
+Here, we use the implementation in the `pancancer-evaluation` parent repo to train mutation prediction classifiers on train/holdout/test sets that are _stratified by cancer type_, and plot the results in the associated notebooks.
+
+A more detailed description of the results and takeaways can be found in the manuscript linked above.
+
+## Repository layout
+
+```
+01_stratified_classification
+|-- nbconverted: notebooks for plotting results, converted to Python scripts (easier to read/review as plain text)
+|-- scripts: scripts for running experiments (locally and on Slurm cluster)
+|-- lasso_range_all_optimizers.ipynb: plot results across all cancer genes (Figures 1C/D and 3A/B)
```

Comment:
Would you consider numbering these scripts so they're easier to walk through? I guess it's tricky when Figure 1 and 3 are made together and figure 2 is separate.